src/webview/webview2/mod.rs

```diff
@@ -311,6 +311,54 @@ impl InnerWebView {
       }
     }
 
+    if let Some(on_navigation_started_handler) = attributes.on_navigation_started_handler {
+      unsafe {
+        webview
+          .add_NavigationStarting(
```

Comment:
there is another call to `add_NavigationStarting` below, move this logic there and only run `on_navigation_started_handler` if `navigation_handler` allowed the navigation